controllers/usersignup/usersignup_controller_test.go

```diff
@@ -101,27 +101,13 @@ func TestUserSignupCreateMUROk(t *testing.T) {
 			// then verify that the MUR exists and is complete
 			require.NoError(t, err)
 			require.Equal(t, reconcile.Result{}, res)
-			murs := &toolchainv1alpha1.MasterUserRecordList{}
-			err = r.Client.List(context.TODO(), murs)
-			require.NoError(t, err)
-			require.Len(t, murs.Items, 1)
-			mur := murs.Items[0]
-			require.Equal(t, test.HostOperatorNs, mur.Namespace)
-			require.Equal(t, userSignup.Name, mur.Labels[toolchainv1alpha1.MasterUserRecordOwnerLabelKey])
-			require.Len(t, mur.Spec.UserAccounts, 1)
-			require.Equal(t, userSignup.Spec.OriginalSub, mur.Spec.OriginalSub)
-			assert.Equal(t, "base", mur.Spec.UserAccounts[0].Spec.NSTemplateSet.TierName)
-			assert.Equal(t, []toolchainv1alpha1.NSTemplateSetNamespace{
-				{
-					TemplateRef: "base-dev-123abc1",
-				},
-				{
-					TemplateRef: "base-stage-123abc2",
-				},
-			}, mur.Spec.UserAccounts[0].Spec.NSTemplateSet.Namespaces)
-			require.NotNil(t, mur.Spec.UserAccounts[0].Spec.NSTemplateSet.ClusterResources)
-			assert.Equal(t, "base-clusterresources-654321b", mur.Spec.UserAccounts[0].Spec.NSTemplateSet.ClusterResources.TemplateRef)
-
+			murtest.AssertThatMasterUserRecord(t, "foo", r.Client).
```

Comment:
to properly replace the existing test code, we should list the MURs and expect that the number is `1`, plus do the additional checks...
the same applies also to the other cases

Reply:
just to clarify: I believe I transposed all the additional checks, but maybe I forgot something? And sure, I can check that there's only 1 MasterUserRecord in each test